src/scss/_utilities.scss

```diff
@@ -39,3 +39,9 @@
 .util-zindex-100 {
   z-index: 100;
 }
+
+.util-delist {
```

Comment:
There's a delist mixin that you could pull in here from `_tools.mixins.general.scss`

Reply:
Nice! I'll use that instead